inference/core/env.py

```diff
@@ -12,8 +12,25 @@
 # The project name, default is "roboflow-platform"
 PROJECT = os.getenv("PROJECT", "roboflow-platform")
 
-# Allow numpy input, default is True
-ALLOW_NUMPY_INPUT = str2bool(os.getenv("ALLOW_NUMPY_INPUT", True))
+# Allow numpy input, default is False
+ALLOW_NUMPY_INPUT = str2bool(os.getenv("ALLOW_NUMPY_INPUT", False))
+ALLOW_URL_INPUT = str2bool(os.getenv("ALLOW_URL_INPUT", True))
+ALLOW_NON_HTTPS_URL_INPUT = str2bool(os.getenv("ALLOW_NON_HTTPS_URL_INPUT", True))
+ALLOW_URL_INPUT_WITHOUT_FQDN = str2bool(os.getenv("ALLOW_URL_INPUT_WITHOUT_FQDN", True))
```

Comment:
Should we make the defaults for `ALLOW_NON_HTTPS_URL_INPUT` and `ALLOW_URL_INPUT_WITHOUT_FQDN` False?

Reply:
👍